app/views/ipAllowlist/ChangeIpAllowlistSuccessView.scala.html

```diff
@@ -39,40 +39,40 @@
 ) {
 
   <div class="ip-allowlist">
-    <div class="govuk-box-highlight">
-      <h1 class="govuk-heading-l govuk-highlight-text ">
+    <div class="govuk-panel govuk-panel--confirmation">
+      <h1 class="govuk-panel__title">
         @if(app.ipAllowlist.allowlist.isEmpty) {
           Your IP allow list is active
         } else {
           IP allow list changes active
         }
       </h1>
-      <p class="govuk-body govuk-highlight-text ">
-        API requests can only be made from these IP addresses
-      </p>
+      <div class="govuk-panel__body">
+          API requests can only be made from these IP addresses
+      </div>
+      <div class="govuk-label--m">
+      </div>
```

Comment:
Do you need this for layout purposes.  It looks like an empty div.

Reply:
Yes Andy. i guess we need it. its like that in prototype.